src/projects/javascript-calculator/index.jsx

```diff
@@ -98,10 +113,12 @@ class Calculator extends React.Component {
           formula: e.target.value !== '0' ? e.target.value : ''
         });
       } else {
+        const currPlusTarget = (this.state.currentVal + e.target.value);
         this.setState({
           currentVal:
             this.state.currentVal === '0' ||
-            isOperator.test(this.state.currentVal)
+            isOperator.test(this.state.currentVal) &&
+            !negativeException(currPlusTarget)
```

Comment:
`!negativeException(currPlusTarget)` always will be true, because of e.target.value is a number in this case.

Reply:
Thank you.